src/stories/Resort/Resort.stories.js

```diff
@@ -73,8 +73,8 @@ export const Resort = () => {
   };
 
   return (
-    <MemoryRouter initialEntries={['resorts/pipedream']}>
-      <Route exact path="resorts/:urlSegment">
+    <MemoryRouter initialEntries={['resort/pipedream']}>
+      <Route exact path="resort/:urlSegment">
```

Comment:
```suggestion
      <Route exact path="resorts/:urlSegment">
```

It's good practice to use plurals for parent segments